hack/build-manifests.sh

```diff
@@ -35,7 +35,6 @@ for arg in $args; do
     sed -e "s/{{ master_ip }}/$master_ip/g" \
         -e "s/{{ primary_nic }}/$primary_nic/g" \
         -e "s/{{ primary_node_name }}/$primary_node_name/g" \
-        -e "s/{{ docker_tag }}/$docker_tag/g" \
-        -e "s/{{ docker_prefix }}/$docker_prefix/g" \
+        -e "/image:/ s#\(.*image:\).*/\(.*\):.*#\1 ${docker_prefix}/\2:${docker_tag}#" \
```

Comment:
I think this is too broad of a search/replace. It could have some confusing consequences in the future. 

Can we limit this to kubevirt related images?

Reply:
Yes - I actually consider dropping this patch in general and use the templating mechanism we had before.

After all the templates themselfes are not an issue (in this patchset), it's rather that I want to drop clster specific things. But the images are not such a thing.

So let me think about it :)

Reply:
Dropped this patch,